packages/libmuparser/build.sh

```diff
@@ -0,0 +1,13 @@
+TERMUX_PKG_HOMEPAGE="https://beltoforion.de/en/muparser"
+TERMUX_PKG_DESCRIPTION="An extensible high performance math expression parser \
+library written in C++"
+TERMUX_PKG_GROUPS="science"
+TERMUX_PKG_LICENSE="BSD 2-Clause"
+TERMUX_PKG_LICENSE_FILE="LICENSE"
```

Comment:
This line shouldn't be needed, LICENSE is one of the filenames [termux_step_install_license looks for](https://github.com/termux/termux-packages/blob/master/scripts/build/termux_step_install_license.sh#L35-L38)